TheMovieDb/Helpers/Constants.swift

```diff
@@ -24,6 +24,7 @@ struct Constants {
     static let titleInitialTableView = "Show Movies"
     static let searchBarPlaceholder = "Search for movies or person"
     static let alertButton = "Dismiss"
+    static let alertEmptyReviews = "Reviews Not Found"
```

Comment:
Remember that if you haver Localizable files, you can move these titles to the file and only get the reference.

Reply:
Yes, I'm going to implement a localizable file in the next PR.